rev-connect-api/pom.xml

```diff
@@ -27,7 +27,7 @@
 		<url/>
 	</scm>
 	<properties>
-		<java.version>17</java.version>
+		<java.version>21</java.version>
```

Comment:
we should probably discuss changing java versions as a group

Reply:
It looks like this was done from @sarangi123 first commit, but I'm not sure why it was changed.